### ISSUE TITLE
Exit code 0 with clojure.test Errors should fail

### ISSUE THREAD

**Issue by AdamClements** [marick/Midje#304]
I have mixed clojure.test and midje facts, when the clojure.tests ERROR, midje's test runner thinks everything has passed and returns an exit code 0. I've ended up deploying failing code from my CI server thanks to this bug, as these particular test failures are only visible by examining the logs.

**Comment by blandinw** [comment]
Where are we on this?

**Comment by jlangr** [comment]
Experienced the same thing, ended up deploying failing code from my CI server thanks to this bug.

Also curious where things are on this.

**Comment by marick** [comment]
I'm nowhere on this yet. But let me look at it for a bit.

**Comment by marick** [comment]
Here's what I did:

* Cloned https://github.com/marick/sample-midje-project.git
* Added this `deftest`:
```clojure
    (deftest foo
      (is (= 1 2)))
```
* `lein midje` with these results:

```
        >>> Output from clojure.test tests:
    
    FAIL in (foo) (t_core.clj:7)
    expected: (= 1 2)
      actual: (not (= 1 2))
    
    1 failures, 0 errors.
    >>> Midje summary:
    No facts were checked. Is that what you wanted?
    Subprocess failed
    652 $ echo $?
    1
```

Can any of you reproduce your problem with that sample project?

I've seen the same behavior with Leiningen 2.3.3 and 2.5.0. I used Midje 1.6.3 and 1.7.0-SNAPSHOT

**Comment by marick** [comment]
Oh, clojure.test ERROR. I hate the distinction between error and failure, think it was one of the two big mistakes in the original sunit (Smalltalk, which let to junit).

(The other was putting the expected value on the left. Beck agreed with me when I put it to him, said it was on the left because of a bug.)

**Comment by blandinw** [comment]
Thanks @marick

**Comment by jlangr** [comment]
thanks Brian!

**Comment by marick** [comment]
This is in 1.7.0-SNAPSHOT, which also has Flare output for better reporting of collection differences. I suppose I could cut a 1.6.5 with just this fix in it, but I'd rather put a 1.7.0-alpha1 on Saturday. Let me know if that's acceptable.
